Makefile

```
@@ -48,3 +48,23 @@ megalint:  ## Run the mega-linter.
 		-v /var/run/docker.sock:/var/run/docker.sock:rw \
 		-v $(shell pwd):/tmp/lint:rw \
 		oxsecurity/megalinter:v7
+
+.PHONY: run
+run:
+	poetry run uvicorn src.main:app --reload --port 5010
+
+.PHONY: docker-build
+docker-build:
+	docker build -t cir-converter-service .
+
+.PHONY: docker-run
+docker-run:
+	docker run -p 5010:5010 cir-converter-service
```

I wonder if we could give the container a name here - this would help to simplify the `docker-stop` command a bit further on then? 

```suggestion
	docker run --name cir-converter-service -p 5010:5010 cir-converter-service
```

---

Just found a simpler solution using start and stop commands, we can discuss in a call.